pydantic_ai_slim/pydantic_ai/agent/__init__.py

```diff
@@ -441,6 +442,7 @@ def iter(
         usage: _usage.RunUsage | None = None,
         infer_name: bool = True,
         toolsets: Sequence[AbstractToolset[AgentDepsT]] | None = None,
+        builtin_tools: Sequence[AbstractBuiltinTool] | None = None,
```

Comment:
Can you please add this to the new `run_stream_events` as well? Here and in the Temporal and DBOS agent classes.